.github/workflows/test-e2e.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches: [master]
 
 jobs:
-  build:
+  e2e:
```

Comment:
Since our tests are not only e2e test, would it be better to use name "ci/test"?

Reply:
agree +1